acquisitions/src/main/resources/thunderjet/consortia/features/pieces-api-test-for-cross-tenant-envs.feature

```diff
@@ -0,0 +1,536 @@
+    * def createCirculationRequest = read('classpath:thunderjet/mod-orders/reusable/create-circulation-request.feature')
+    * def createUserGroup = read('classpath:thunderjet/mod-orders/reusable/user-init-data.feature@CreateGroup')
+    * def setUserPatronGroup = read('classpath:thunderjet/mod-orders/reusable/user-init-data.feature@SetUserPatronGroup')
+
+    * def fundId = callonce uuid
+    * def orderId = callonce uuid
+    * def poLineId = callonce uuid
+    * def titleId = callonce uuid
+    * def pieceId = callonce uuid
+
+    * callonce createOrder { id: #(orderId) }
+    * callonce createOrderLine { id: #(poLineId), orderId: #(orderId), isPackage: True }
+    * callonce createTitle { titleId: #(titleId), poLineId: #(poLineId) }
+
+
+  Scenario: Check ShadowInstance, Holding and Item created in member tenant when creating piece
+    # Create a new piece
+    * set minimalPiece.id = pieceId;
+    * set minimalPiece.titleId = titleId;
+    * set minimalPiece.poLineId = poLineId;
+    * set minimalPiece.locationId = universityLocationsId;
+    * set minimalPiece.receivingTenantId = universityTenant;
+    Given path 'orders/pieces'
+    And param createItem = true
+    And request minimalPiece
+    When method POST
+    Then status 201
+    And match response.id == '#(pieceId)'
+    And match response.poLineId == '#(poLineId)'
+    And match response.titleId == '#(titleId)'
+    And match response.itemId == '#present'
+    And match response.holdingId == '#present'
+    And match response.receivingTenantId == '#(universityTenant)'
+    And def holdingId = response.holdingId
+
+    # Check the created holding record in specified tenant
+    Given path '/holdings-storage/holdings/', holdingId
+    And header x-okapi-tenant = universityTenant
+    When method GET
+    Then status 200
+    And match response.instanceId == '#present'
+    And def instanceId = response.instanceId
+
+    # Check the created shared instance in specified tenant
+    Given path '/instance-storage/instances', instanceId
+    And header x-okapi-tenant = universityTenant
+    When method GET
+    Then status 200
+
+    # Check the created item in specified tenant
+    Given path '/inventory/items'
+    And header x-okapi-tenant = universityTenant
+    And param query = 'holdingsRecordId=' + holdingId
+    When method GET
+    Then status 200
+    And match response.totalRecords == 1
+    And match response.items[0].holdingsRecordId == '#(holdingId)'
+    And match response.items[0].purchaseOrderLineIdentifier == '#(poLineId)'
+
+
+  Scenario: Check Holding and Item updated in member tenant when updating piece without itemId and deleteHolding=true
+    # Get existing holdingId and itemId in specified tenant
+    Given path 'orders/pieces', pieceId
+    When method GET
+    Then status 200
+    And match response.itemId == '#present'
+    And match response.holdingId == '#present'
+    And def oldItemId = response.itemId
+    And def oldHoldingId = response.holdingId
```

Comment:
we also should specify central tenant here as header

Reply:
Added to other occurrences too